<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>func3</class>
 <widget class="QMainWindow" name="func3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>布局</string>
    </property>
    <addaction name="action_sudoku"/>
    <addaction name="action_v"/>
    <addaction name="action_h"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="w_1">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(219, 255, 252);</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::DockWidgetArea::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>1</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9"/>
  </widget>
  <widget class="QDockWidget" name="w_2">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="windowTitle">
    <string>2</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_11"/>
  </widget>
  <widget class="QDockWidget" name="w_3">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 0);</string>
   </property>
   <property name="windowTitle">
    <string>3</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_12"/>
  </widget>
  <widget class="QDockWidget" name="w_4">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 127);</string>
   </property>
   <property name="windowTitle">
    <string>4</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_13"/>
  </widget>
  <widget class="QDockWidget" name="w_5">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="windowTitle">
    <string>5</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_14"/>
  </widget>
  <widget class="QDockWidget" name="w_6">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="windowTitle">
    <string>6</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_15"/>
  </widget>
  <widget class="QDockWidget" name="w_7">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 127);</string>
   </property>
   <property name="windowTitle">
    <string>7</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_16"/>
  </widget>
  <widget class="QDockWidget" name="w_8">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);</string>
   </property>
   <property name="windowTitle">
    <string>8</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_18"/>
  </widget>
  <widget class="QDockWidget" name="w_9">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="windowTitle">
    <string>9</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_19"/>
  </widget>
  <action name="action_sudoku">
   <property name="text">
    <string>九宫格</string>
   </property>
  </action>
  <action name="action_v">
   <property name="text">
    <string>垂直</string>
   </property>
  </action>
  <action name="action_h">
   <property name="text">
    <string>水平</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
